modules/nlu/src/backend/utterance/augmenter.ts

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash'
+
+interface Augmentation {
```

Comment:
well this is what I meant by SlotExample, we should call this SlotAugmentation with slotName instead of variableName